diesel/src/sqlite/expression/operators.rs

```diff
@@ -0,0 +1,5 @@
+use crate::sql_types::Bool;
+use crate::sqlite::Sqlite;
+
+__diesel_infix_operator!(Is, " IS ", ConstantNullability Bool, backend: Sqlite);
```

Comment:
What happens if you compare a null value to a text value?

Reply:
This behaves identically to the `=` operator, except that `NULL` is treated as a normal value.

I will add the document.

Reply:
@pksunkara Added.

Reply:
The IS and IS NOT operators work like = and != except when one or both of the operands are NULL. In this case, if both operands are NULL, then the IS operator evaluates to 1 (true) and the IS NOT operator evaluates to 0 (false). If one operand is NULL and the other is not, then the IS operator evaluates to 0 (false) and the IS NOT operator is 1 (true). It is not possible for an IS or IS NOT expression to evaluate to NULL.

Reply:
The IS and IS NOT operators work like = and != except when one or both of the operands are NULL. In this case, if both operands are NULL, then the IS operator evaluates to 1 (true) and the IS NOT operator evaluates to 0 (false). If one operand is NULL and the other is not, then the IS operator evaluates to 0 (false) and the IS NOT operator is 1 (true). It is not possible for an IS or IS NOT expression to evaluate to NULL.

Reply:
I would prefer to include something more in line with this description to the documentation, so that it explicitly states the handling of `NULL` values.

Reply:
@weiznich Added.